readme.md

```diff
@@ -1,191 +1,303 @@
+* employ tools that automatically migrate legacy code or assist in migration
+* not ossify poor choices and be honest that we can't make such guarantees
+
+Popular languages are maintained through incredible amounts of funding from
+various entities; we do not see, nor seek, this happening for us.
+Alternatively, there are a number of languages that require unhealthy amounts
+of free labour from a few, we're not interested in that either. Instead as is
+described this language will focus on practitioners able to affect their tools
+and community.
+
+</details>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Koch
+
+``koch`` is the build tool used to build various parts of Nim and to generate
+documentation, among other things. The ``koch`` tool can also
+be used to run the Nim test suite.
+
+<details>
+<summary>Show</summary>
+
+Assuming that you added Nim's ``bin`` directory to your PATH, you may execute
+the tests using ``./koch tests``. The tests take a while to run, but you
+can run a subset of tests by specifying a category (for example
+``./koch tests cat async``).
+
+For more information on the ``koch`` build tool please see the documentation
+within the [doc/koch.rst](doc/koch.rst) file.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+</details>
+
+## FAQ
+<details>
+<summary class="blue">What's the rational for this fork?</summary>
+We love the original vision behind nim-lang; we believe that an element of the
+vision was lost. We simply believe a different development process was needed to that which was being used and decided that the best way to see this vision materialize was to enforce that process ourselves.
+</details>
+
+<details>
+<summary class="blue">What are you going to do now?</summary>
+For the moment, please see our [projects board](https://github.com/nim-works/nimskull/projects) and [direction](#direction) for more information. We envisage great things; however, all great things come with time, and we have a large foundation that was never properly solidified.
+</details>
+
+<details>
+<summary class="blue">Will this break my Nim code?</summary>
+Maybe. Many experienced users will know that a lot of current code 'works' because of various hacks, or create hacks themselves to make code work. See <a href="https://github.com/nim-works/nimskull/issues/8">#8</a> and the <a href="https://github.com/nim-works/nimskull#direction">direction</a> for more on this.
+</details>
+
+<details>
+<summary class="blue">Will you plan to merge Nim upstream changes?</summary>
+Generally? No. There will likely be exceptions to this. Our current focus remains on solidifying the foundations before moving forward.
+</details>
+
+<details>
+<summary class="blue">Can I help somehow?</summary>
+Presently we're very interested in people contributing than to growing our community; so if you would like to contribute, a good start is to help haxscramper with the language spec tests

```

Comment:
> a good start is to help haxscramper with the language spec tests 

Don't redirect to me, redirect to #10 for to-do, or #28 for examples on how this should be done.